### ISSUE TITLE
Type definitions for project

### ISSUE THREAD

**Issue by FranciscoKloganB** [FatehAK/vite-plugin-image-optimizer#12]
### Describe the Bug

When I import `ViteImageOptimizer }` in `nuxt.config.ts` on a TypeScript based project
Then a warning about missing type declarations occurs

### Steps to reproduce

Create new nuxt project
Install plugins and dependencies (sharp, svgo)

Add to `nuxt.config.ts`

```ts
// `Could not find a declaration file for module 'vite-plugin-image-optimizer'`
import { ViteImageOptimizer } from 'vite-plugin-image-optimizer'
```

Results in the error above, no types exist for lib.

### System Info

```shell
Windows 11 (but same result in Ubuntu 22.04 and MacOS latest)

 System:
    OS: Windows 10 10.0.22621
    CPU: (16) x64 AMD Ryzen 7 5800X 8-Core Processor
    Memory: 37.36 GB / 63.93 GB
  Binaries:
    Node: 18.12.1 - C:\Program Files\nodejs\node.EXE
    npm: 9.5.1 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.22621.963.0), Chromium (110.0.1587.49)
    Internet Explorer: 11.0.22621.1
  npmPackages:
    @vitejs/plugin-vue: ^4.0.0 => 4.0.0
    vite: ^4.1.4 => 4.1.4
    vite-plugin-image-optimizer: ^1.0.8 => 1.0.8
    vite-tsconfig-paths: ^4.0.5 => 4.0.5
    vitest: ^0.28.5 => 0.28.5
```

### Used Package Manager

pnpm

### Validations

- [X] Read the [docs](https://github.com/FatehAK/vite-plugin-image-optimizer#readme).
- [X] Check that there isn't [already an issue](https://github.com/FatehAK/vite-plugin-image-optimizer/issues) that reports the same bug to avoid creating a duplicate.

**Comment by FatehAK** [comment]
Thanks for raising this issue. Type definitions have been added to the project now. Please update the package to `v1.0.10`.

**Comment by FatehAK** [comment]
I've published `v1.1.0` with full TS typings please check with that version.
